src/sidebar/store/modules/selection.js

```diff
@@ -97,7 +120,22 @@ function init(settings) {
 }
 
 const update = {
-  CLEAR_SELECTION: function() {
+  CLEAR_SELECTION: function(state) {
+    let selectedTab = state.selectedTab;
+    if (selectedTab === uiConstants.TAB_ORPHANS) {
+      selectedTab = uiConstants.TAB_ANNOTATIONS;
+    }
+    const tabSettings = setTab(state.selectedTab, selectedTab);
+    return Object.assign(
```

Comment:
Pro-tip: If you want to merge two objects together you can do that a bit more simply in new code using "object spread" syntax:

```js
return { filterQuery: null, selectedAnnotationMap: null, ...tabSettings };
```